src/transformers/training_args.py

```diff
@@ -80,6 +80,7 @@ class OptimizerNames(ExplicitEnum):
     ADAMW_TORCH_XLA = "adamw_torch_xla"
     ADAMW_APEX_FUSED = "adamw_apex_fused"
     ADAFACTOR = "adafactor"
+    ADAMW_BNB = "adamw_bnb"
```

Comment:
Should we name it `adam_8_bit` ?

Reply:
@sgugger I don't have any preferences. Do you want `adam_8_bit` or `adamw_8_bit` to be consistent with the other strings?

Reply:
Definitely `adamw_` , for the prefix part since all the others use that.

Then we usually add the project and finally specific bit, thus I propose:

`adamw_bnb_8bit`

since we are likely to see new optimizers from BNB down the road.

Reply:
Renamed to `adamw_bnb_8bit`

Reply:
I think `adamw_bnb_8bit` is good for now. I will release more specialized optimizers that are faster than the current one, but currently, people only use dynamic quantization anyway, so I do not think there will be different 8-bit optimizers for a while.